test/JDBC/expected/Test-sp_addrole-vu-verify.out

```diff
@@ -92,6 +99,15 @@ WHERE orig_username = 'sp_addrole_r3'
 GO
 ~~START~~
 varchar#!#char#!#nvarchar#!#nvarchar
-master_sp_addrole_r3#!#R#!#sp_addrole_r3#!#master
+~~END~~
+
+
+SELECT rolname, type, orig_username, database_name
+FROM sys.babelfish_authid_user_ext
+WHERE orig_username = 'SP_ADDROLE_R3'
+GO
+~~START~~
+varchar#!#char#!#nvarchar#!#nvarchar
+master_sp_addrole_r3#!#R#!#SP_ADDROLE_R3#!#master
 ~~END~~
 
```

Comment:
Please add a line here to select from `sys.database_principals` and see if the result is the same as SQL server

Reply:
Fixed in next revision by adding check point sys.database_principals to see the result of added role into the database.